### ISSUE TITLE
[comp:select] 配置了 labelKey 之后无法正常过滤筛选

### ISSUE THREAD

**Issue by Jouryjc** [IDuxFE/idux#750]
- [x] I have searched the [issues](https://github.com/IDuxFE/idux/issues) of this repository and believe that this is not a duplicate.
          
### Reproduction link
[https://codesandbox.io/s/smoosh-bush-qyc9d?file=/src/Demo.vue](https://codesandbox.io/s/smoosh-bush-qyc9d?file=/src/Demo.vue)

### Steps to reproduce
1. 配置 select 的 valueKey、labelKey 和 searchable

```vue
<IxSelect
      v-model:value="value"
      :options="options"
      :searchable="searchableValue"
      placeholder="Searchable"
      labelKey="name"
      valueKey="id"
      @change="onChange"
    >
```
2. 输入框内输入搜索

### What is expected?
能够正常过滤

### What is actually happening?
不能正常过滤

### Environment Info 

```shell
 Browsers:
    Chrome: 98.0.4758.82
    Edge: Spartan (44.17763.831.0)
    Internet Explorer: 11.0.17763.771
  npmPackages:
    vue: ^3.2.6 => 3.2.30
```

**Comment by danranVm** [comment]
tree 也有这种情况

**Comment by danranVm** [comment]
closed #869
